docker-compose.yaml

```diff
@@ -10,14 +10,23 @@
 version: '3.8'
 services:
   model-catalog:
-    image: ghcr.io/kubeflow/model-registry/server:latest
+    image: localhost:5001/kubeflow/model-registry/server:main
```

Comment:
Should this have been left pointing to a remote registry? Just checking if this was left over from testing